src/App.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { BrowserRouter, Switch, Route } from "react-router-dom";
+import {Helmet} from 'react-helmet';
+
+// import application components
+import ContactForm from "./ContactForm/ContactForm";
+
+export default class App extends React.Component {
+
+  render() {
+    return (
+      <>
+        <Helmet>
+            <meta charSet="utf-8" />
+            <title>Resume Builder</title>
+        </Helmet>
+        <BrowserRouter>
+          <Switch>
+            <Route path='/' exact component={ContactForm} />
+          </Switch>
```

Comment:
I'm not too experienced with Router but is Switch needed here for one route?

Reply:
No .. it was left over from a copy and paste from another project. Normally I'd drop the switch or i'd add a 404 page just in case the user entered the wrong url. For today I was focused on getting things up but good catch.